transforms/__testfixtures__/underscore-to-lodash-native.output.js

```diff
@@ -0,0 +1,5 @@
+// Test comment
+import {find} from 'lodash';
+[1, 2].forEach(num => num);
+const test = [1, 2, 3];
+const result = find(test, 2);
```

Comment:
👍 I bet there's a bunch more we could convert to native pretty easily. How do you feel about adding a test for every underscore method so we can make sure that they are all converted in the best possible way?

e.g.

```js
const test = [1, 2, 3];
const result = _.max(test);
```

could be:

```js
const test = [1, 2, 3];
const result = Math.max(...test);
```